epregressions/tk_window.py

```diff
@@ -638,6 +643,24 @@ def idf_select_all(self):
             self.active_idf_listbox.insert(END, idf)
         self.idf_refresh_count_status()
 
+    def idf_select_all_except_long_runs(self):
```

Comment:
Add in a new way to select all but the longest running files.  This can take tons of time off the run just by skipping these few files.